From: Felix Fietkau <nbd@nbd.name>
Date: Tue, 14 Jan 2025 11:42:25 +0100
Subject: [PATCH] backport-include: fix linux/acpi_amd_wbrf.h inclusion

Fix building for kernel >= 6.8 by adjusting incorrect guard usage,
otherwise an #include_next header is masked and compilation will fail
for net/mac80211/wbrf.c in the mac80211 kernel module.

Fixes: 52cdcaab ("backport-include: backport linux/acpi_amd_wbrf.h")
Reported-by: Tony Ambardar <itugrok@yahoo.com>
Signed-off-by: Felix Fietkau <nbd@nbd.name>
---

--- a/backport-include/linux/acpi_amd_wbrf.h
+++ b/backport-include/linux/acpi_amd_wbrf.h
@@ -4,8 +4,8 @@
  * Copyright (C) 2023 Advanced Micro Devices
  */
 
-#ifndef _ACPI_AMD_WBRF_H
-#define _ACPI_AMD_WBRF_H
+#ifndef __BACKPORT_ACPI_AMD_WBRF_H
+#define __BACKPORT_ACPI_AMD_WBRF_H
 
 #if LINUX_VERSION_IS_GEQ(6,8,0)
 #include_next <linux/acpi_amd_wbrf.h>
@@ -83,4 +83,4 @@ int amd_wbrf_unregister_notifier(struct
 }
 
 #endif /* >=6,8,0 */
-#endif /* _ACPI_AMD_WBRF_H */
+#endif /* __BACKPORT_ACPI_AMD_WBRF_H */
